web/src/components/navbar/NavSideBarUserSettings.tsx

```diff
@@ -28,38 +36,62 @@ export const NavSideBarUserSettings = (): ReactElement => {
           id="user-profile-header"
         >
           <div className="margin-y-2">
-            <h4 className="flex flex-align-center text-primary">
-              <Icon className="margin-right-1 usa-icon--size-3">account_circle</Icon>
-              <span>{userName}</span>
+            <h4
+              className={`flex flex-align-center text-${
+                state.isAuthenticated == "TRUE" ? "primary" : "base"
+              }`}
+            >
+              <Icon className="margin-right-1 usa-icon--size-3">
+                {state.isAuthenticated == "TRUE" ? "account_circle" : "help"}
+              </Icon>
+              <span>
+                {state.isAuthenticated == "TRUE" ? userName : Config.navigationDefaults.navBarGuestText}
+              </span>
             </h4>
           </div>
         </AccordionSummary>
         <AccordionDetails>
           <div className="margin-left-2 margin-bottom-2">
-            <div className="margin-bottom-2">
-              <button
-                className={`clear-button override-text-base`}
-                onClick={(event) => {
-                  event.preventDefault();
-                  analytics.event.account_menu_myNJ_account.click.go_to_myNJ_home();
-                  window.open(process.env.MYNJ_PROFILE_LINK || "", "_ blank");
-                }}
-              >
-                {Config.navigationDefaults.myNJAccountText}
-              </button>
-            </div>
-            <div className="margin-bottom-2">
-              <Link href="/profile" passHref>
-                <button
-                  className="clear-button override-text-base"
+            {state.isAuthenticated == "TRUE" ? (
+              <>
+                <div className="margin-bottom-2">
+                  <Button
+                    style="tertiary"
+                    onClick={(event) => {
+                      event.preventDefault();
+                      analytics.event.account_menu_myNJ_account.click.go_to_myNJ_home();
+                      window.open(process.env.MYNJ_PROFILE_LINK || "", "_ blank");
+                    }}
+                  >
+                    <span className="text-base">{Config.navigationDefaults.myNJAccountText}</span>
+                  </Button>
+                </div>
+                <div className="margin-bottom-2">
+                  <Link href="/profile" passHref>
+                    <Button
+                      style="tertiary"
+                      onClick={() => {
+                        analytics.event.account_menu_my_profile.click.go_to_profile_screen();
+                      }}
+                    >
+                      <span className="text-base">{Config.navigationDefaults.profileLinkText}</span>
+                    </Button>
+                  </Link>
+                </div>
+              </>
+            ) : (
```

Comment:
can we refactor this component to call functions or components? This is hard to read, would be way easier if it was like `{state.isAuthenticated == "TRUE" ? renderSomeDescriptiveName() : renderAnotherThing()}`